v1/python-sdk/tutorials/automl-with-azureml/forecasting-orange-juice-sales/auto-ml-forecasting-orange-juice-sales.ipynb

```diff
@@ -687,6 +715,7 @@
    ]
   },
   {
+   "attachments": {},
```

Comment:
Why do we have this tag? Was it generated by a new version of jupyter?

Reply:
Not sure.